changelog/unreleased/4404

```diff
@@ -0,0 +1,9 @@
+Enhancement: Improvements in remove dialog
+
+The custom dialog has been modified when the file that is going to be deleted has thumbnail.
+Also, when removing files in multiple selection, the number of elements that are going to be removed is displayed in the dialog.
+
+https://github.com/owncloud/android/issues/4377
+https://github.com/owncloud/android/issues/4342
+https://github.com/owncloud/android/pull/4348
+https://github.com/owncloud/android/pull/4404
```

Comment:
Let's try to keep a bit of order here. If you added the new PR in second place, let's do the same with the issue, otherwise they won't be corresponding

Reply:
Changed the order.